ibm/resource_ibm_container_alb_create.go

```diff
@@ -0,0 +1,198 @@
+// Copyright IBM Corp. 2017, 2021 All Rights Reserved.
+// Licensed under the Mozilla Public License v2.0
+
+package ibm
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	v1 "github.com/IBM-Cloud/bluemix-go/api/container/containerv1"
+)
+
+func resourceIBMContainerAlbCreate() *schema.Resource {
+	return &schema.Resource{
+		Create:   resourceIBMContainerClassicAlbCreate,
+		Read:     resourceIBMContainerALBRead,
+		Update:   resourceIBMContainerALBUpdate,
+		Delete:   resourceIBMContainerALBDelete,
+		Importer: &schema.ResourceImporter{},
+		Timeouts: &schema.ResourceTimeout{
+			Create: schema.DefaultTimeout(10 * time.Minute),
+			Update: schema.DefaultTimeout(10 * time.Minute),
+		},
+
+		Schema: map[string]*schema.Schema{
+			//post req
+			"enable": {
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Default:     true,
+				Description: "If set to true, the ALB is enabled by default.",
+			},
+			"ingress_image": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				ForceNew:    true,
+				Description: "The type of Ingress image that you want to use for your ALB deployment.",
+			},
+			"ip": {
+				Type:     schema.TypeString,
+				Optional: true,
+				//ForceNew:    true,
+				Description: "The IP address that you want to assign to the ALB.",
+			},
+			"nlb_version": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				ForceNew:    true,
+				Description: "The version of the network load balancer that you want to use for the ALB.",
+			},
+			"alb_type": {
+				Type:     schema.TypeString,
+				Required: true,
+				//ForceNew:    true,
+				Description: "The type of ALB that you want to create.",
+			},
+			"vlan_id": {
+				Type:     schema.TypeString,
+				Required: true,
+				//ForceNew:    true,
+				Description: "The VLAN ID that you want to use for your ALBs.",
+			},
+			"zone": {
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: "The zone where you want to deploy the ALB.",
+			},
+			"cluster": {
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: "The ID of the cluster that the ALB belongs to.",
+			},
+
+			//response
+			"alb_id": {
+				Type: schema.TypeString,
+				//Required:    false,
+				Computed:    true,
+				ForceNew:    true,
+				Description: "The ID of the application load balancer (ALB).",
+			},
+			"name": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "ALB name",
+			},
+			"disable_deployment": {
```

Comment:
All these attributes are part of Enable an existing ALB in a clusterAPI the existing resource already support (resource_ibm_container_alb) it why we need as part of this resource schema as optional ?
I think all these should be only computed?